setup.py

```diff
@@ -47,6 +47,7 @@
         "rtree>=1.2.0",
         "graphviz>=0.20",
         "matplotlib>=3.0.0",
+        "curobo @ git+https://github.com/StanfordVL/curobo@main",
```

Comment:
@cgokmen Apparently CI doesn't like this - tests are unable to run. Any idea how to fix this?

Reply:
@cgokmen @hang-yin @cremebrule Actually Arpi is having trouble installing this with the same error message. Could you guys help me try?

git clone SVL's copy of curobo or the official one
and then run `pip install -e . --no-build-isolation`